statsmodels/stats/tests/test_moment_helpers.py

```diff
@@ -87,20 +87,34 @@ def test_moment_conversion():
         #assert_equal(mvsk2mnc(mc2mvsk(mom[0])), mc2mnc(mom[0]))
 
 
+    multidimention_test = [np.array([[   5.,   10.,    1.],
+       [   5.,   10.,    1.],
+       [   5.,   10.,    1.],
+       [  80.,  310.,    4.]]), np.random.randint(0, 100, 12).reshape(4,3).astype(np.float32)]
+
+    for test_vals in multidimention_test:
+        assert_equal(cum2mc(mnc2cum(mc2mnc(test_vals).T).T).T.astype(np.float32), test_vals)
+        assert_equal(cum2mc(mc2cum(test_vals).T).T.astype(np.float32), test_vals)
+        assert_equal(mvsk2mc(mc2mvsk(test_vals).T).T.astype(np.float32), test_vals)
+
```

Comment:
another test here could compare the outcome one of the function in vectorized form with the list comprehension as I did in issue #2928